python/Ganga/Runtime/bootstrap.py

```diff
@@ -1216,10 +1216,16 @@ def launch_IPython(local_ns, args, error_handler, ganga_prompt):
 
         import IPython
 
+	ipver = IPython.__version__
```

Comment:
Possibly, I think we're allowed to stick in tabs here by accident (!) I'll check privately.